benchmarks/benchmark1.ipynb

```diff
@@ -1001,13 +1001,57 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>We will need to remember to update the link to the upload pages when we have those updated. Keep that link for now as it at least links to the old-old upload method.</li><li>"This data can be uploaded directly to the website ..." is obviously not correct, but this will need to updated across all the notebooks when we create the then upload documentation. Maybe remove this for now.</li><li>"... in a file named needs to be ..." doesn't make sense</li><li>"This CSV file" -&gt; "The CSV file"</li><li>"CSV file" could be a link to the actual CSV formatting spec on wikipedia or something</li><li>"we shall endeavor" -&gt; the PFHub team or the PFHub operation</li><li>"The same data as" -&gt; "The same data files as in"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Agreed, except that "this CSV file" is correct.